config/routes.rb

```diff
@@ -3,6 +3,8 @@
 Rails.application.routes.draw do
   root 'static_pages#home'
   get '/admin', to: 'static_pages#admin_home'
+  get '/admin/products/:id/restore', to: 'products#restore', as: 'product_restore'
```

Comment:
Are you sure that it's good to use `get` method there as we change the product in database?